js/core/utils/dom.js

```diff
@@ -93,22 +93,10 @@ export const contains = (container, element) => {
     if(!element) {
         return false;
     }
-
-    if(domAdapter.isTextNode(element)) {
-        element = element.parentNode;
-    }
-
-    if(domAdapter.isDocument(container)) {
-        return container.documentElement.contains(element);
-    }
-
     if(isWindow(container)) {
         return contains(container.document, element);
     }
-
-    return container.contains
-        ? container.contains(element)
-        : !!(element.compareDocumentPosition(container) & element.DOCUMENT_POSITION_CONTAINS);
```

Comment:
Why did u removed this code?

Reply:
Because it was made for IE (https://github.com/DevExpress/DevExtreme/pull/13136). Now we does not support IE